sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -36,85 +58,105 @@
-      "allowMultipleMasks": false,
-      "tree": false,
       "components": [
         {
-          "html": "<p><span style=\"background-color:rgb(255,255,255);color:rgba(0,0,0,0.87);\">Please enter your institution information below. You will be able edit these details in your institution profile.</span></p>",
-          "label": "Content",
-          "customClass": "",
+          "label": "HTML",
+          "attrs": [
+            {
+              "attr": "",
+              "value": ""
+            }
+          ],
+          "content": "Institution details",
           "refreshOnChange": false,
+          "customClass": "category-header-large primary-color",
+          "key": "html",
+          "type": "htmlelement",
+          "input": false,
+          "tableView": false,
+          "placeholder": "",
+          "prefix": "",
+          "suffix": "",
+          "multiple": false,
+          "defaultValue": null,
+          "protected": false,
+          "unique": false,
+          "persistent": false,
           "hidden": false,
+          "clearOnHide": true,
+          "refreshOn": "",
+          "redrawOn": "",
           "modalEdit": false,
-          "key": "content",
-          "tags": [],
-          "properties": {},
+          "dataGridLabel": false,
+          "labelPosition": "top",
+          "description": "",
+          "errorLabel": "",
+          "tooltip": "",
+          "hideLabel": false,
+          "tabindex": "",
+          "disabled": false,
+          "autofocus": false,
+          "dbIndex": false,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "widget": null,
+          "attributes": {},
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "custom": "",
+            "customPrivate": false,
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
           "conditional": {
             "show": null,
             "when": null,
-            "eq": "",
-            "json": ""
+            "eq": ""
           },
-          "customConditional": "",
-          "logic": [],
-          "attributes": {},
           "overlay": {
             "style": "",
-            "page": "",
             "left": "",
             "top": "",
             "width": "",
             "height": ""
           },
+          "allowCalculateOverride": false,
+          "encrypted": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "properties": {},
+          "allowMultipleMasks": false,
+          "addons": [],
+          "tag": "p",
+          "id": "ete7bv"
+        },
+        {
+          "html": "<p><span style=\"background-color:rgb(255,255,255);color:rgba(0,0,0,0.87);\">Please enter your institution information below. You will be able edit these details in your institution profile.</span></p>",

```

Comment:
you can create custom class in vue and use here

Reply:
This is a content component and also not part of PR. Not sure why it got added to changes.